layouts/sidebar.html

```diff
@@ -33,6 +33,7 @@ <h4 class="sidebar-title">API v2</h4>
     <li><a href="/v2/zones/records/">Records</a></li>
   </ul></li>
   <li><a href="/v2/contacts/">Contacts</a></li>
+  <li><a href="/v2/secondary-dns/">Secondary DNS</a></li>
```

Comment:
I think we can leave this out for the moment until we launch the feature publicly.

Reply:
Yes, @DXTimer is right. We will share the direct link with customers that we want to test the feature with, but it will not be announced to the general public yet.

Reply:
should we keep the `openapi` change so?

Reply:
Yes, please to keep the OpenAPI. That's our primary source. It is reasonable to not link yet publish in the OpenAPI. The goal is not to hide, but simply to limit exposure until we're ready to advertise it.